maintainers-circle/README.md

```diff
@@ -1,49 +1,76 @@
 # Maintainer's Circle
 
-Placeholder for the Maintainer's Circle subproject [proposed here](https://github.com/cncf/sig-contributor-strategy/issues/1).
+Initially [proposed here](https://github.com/cncf/sig-contributor-strategy/issues/1).
+
+Once a month, we welcome you to a maintainers circle: a peer group that will 
+meet with scheduled topics, ample discussion, and networking time for fellow 
+reviewers, approvers, maintainers, and other core leaders of CNCF projects. This
+is a time for you to stay connected, learn from and with each other, and 
+discuss topics that are close to you. We'll have a 100 person capacity for each 
+event and depending on demand, may re-run the session. The capacity is set so
+this is not a lecture but 
+
+The only contract of the members of each circle is to abide by the CNCF Code of 
+Conduct, be kind, open as you'd like, and committed to hearing others out. 
+Everyone will have a chance to participate in whatever activity is set forth in 
+that meeting. 
 
 ## Goals:
-Bring CNCF project maintainers🔹 together to:
+Bring CNCF project maintainers🔹together to:
```

Comment:
Is this a stray emoji? Doesn't seem like it fits here.

Reply:
its linking to the other maintainer word above but I can use an asterisk instead

Reply:
Oooooooh! 🤦‍♀️ I totally didn't understand that. Yeah maybe an asterisk would be more clear? I know that's less fun but even I missed how the emoji was being used.